ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -792,6 +793,11 @@ static UnsignedLong get_domain(Fork fork, UnsignedLong epoch, int domain_type) {
         .plus(UnsignedLong.valueOf(domain_type));
   }
 
+  /** Extract the bit in ``bitfield`` at position ``i``. */
+  public static int get_bitfield_bit(Bytes bitfield, int i) {
+    return (bitfield.get(i / 8) >> (i % 8)) % 2;
```

Comment:
This doesn't seem to match v0.1 of the spec.  My interpretation is this:

`return (bitfield.get(i / 8) >> (7 - (i % 8))) % 2;`

Reply:
I implemented the one from master and not v0.1 - should I update it to v0.1?